src/main/java/com/codeplanks/home360/listing/ListingCost.java

```diff
@@ -19,16 +19,16 @@
 public class ListingCost {
     @NotNull(message = "Annual rent is required")
     @Field(name = "annual_rent")
-    private BigDecimal annualRent;
+    private Integer annualRent;

```

Comment:
Should this be `Integer` or `Long`? read more, see which fits appropriately and refactor.